src/main/java/org/killbill/billing/plugin/stripe/StripePluginProperties.java

```diff
@@ -215,7 +215,7 @@
         return additionalDataMap;
     }
 
-    public static Map<String, Object> toAdditionalDataMap(final Session session) {
+    public static Map<String, Object> toAdditionalDataMap(final Session session, String pk) {
```

Comment:
Could be `final`. That being said, why is it passed here?

Reply:
Changing to final. I found I needed to pass the pk to the Stripe HPP.